.github/workflows/docker-push-to-registries.yml

```diff
@@ -103,13 +116,18 @@ jobs:
         run: |
           mvn verify --batch-mode -DskipTests -Dgpg.skip=true
 
-      # The existence of this SBOM is context dependent, some teams build workflows may not generate a
-      # SBOM in this fashion hence continue-on-error: true
       - name: Get SBOM from artifact
         uses: actions/download-artifact@v4
+        # Unsure if SBOM exists (for now)
         continue-on-error: true
         with:
-          name: ${{ github.run_number }}-sbom.json
+          name: ${{ github.run_number }}${{ (inputs.PACKAGE_DIRECTORY == '.' && '') || inputs.PACKAGE_DIRECTORY }}.sbom.json
+          path: ${{ inputs.PATH }} 
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3.2.0
+        with:
+          platforms: all
```

Comment:
This step seems to have been duplicated (also appears lines 132-135) - bad merge?

Reply:
resolved: https://github.com/telicent-oss/shared-workflows/pull/45/commits/2a658c19b203ce659049b74f9cf7303e63e30854